util/src/rlp/bytes.rs

```diff
@@ -74,7 +37,9 @@ pub trait ToBytes {
 
 impl <'a> ToBytes for &'a str {
 	fn to_bytes<V: VecLike<u8>>(&self, out: &mut V) {
-		out.vec_extend(self.as_bytes());
+		for byte in self.as_bytes() {
+			out.push(byte.to_owned());
```

Comment:
`for &byte in self.as_bytes()` and no `to_owned` call would make this a little cleaner.